.azure-pipelines.yml

```diff
@@ -49,7 +53,7 @@ stages:
       iosVariableGroup: 'ApplicationTemplate.Distribution.Internal.iOS'
 
 - stage: Publish_Template_Package
-  condition: and(succeeded(), eq(variables['IsLightBuild'], 'false'))
+  condition: and(succeeded(), eq(variables['IsLightBuild'], 'false'), eq(variables['IsReleaseBranch'], 'true'))
```

Comment:
This will not work if the IsReleaseBranch variable gets removed when the project is generated.

Reply:
but that script is also not part of the generated project. That's why I've specified to exclude that variable in the generated project

Reply:
Oh right, GitHub PRs don't show the full file so I missed that!